lib/commands/project/project.js

```diff
@@ -95,6 +95,20 @@ function create(name, options, cb) {
     cloneSkeleton(name, framework, targetDir, function(err) {
       if (err) { return cb(err); }
       emit('Project %s created in %s', name, targetDir);
+
+      findProjectFile(targetDir, null, function(err, projPath) {
+        var projectFile = fs.readFileSync(projPath, 'utf8');
+        var projectJson = JSON.parse(projectFile);
+
+        projectJson.devDependencies['z-schema'] = '^3.12.0';
```

Comment:
Check if a module is already installed. `generate-test` will eventually work for when you generated tests already and now you want to generate more tests based on your new additional paths. in that case, user might've updated the package.json with a newer version of these dependencies. We don't want to override those.